blocks/shared-styles/_children/promo-headline/index.test.jsx

```diff
@@ -76,3 +76,25 @@ describe('PromoHeadline', () => {
     expect(wrapper.find('a').prop('href')).toBe('https://arcxp.com');
   });
 });
+
+it('renders link with link text from web prop when basic equals original', () => {
```

Comment:
Maybe we should set up a test where basic !=original as well?

Reply:
hmm, ok